package.json

```diff
@@ -17,12 +19,12 @@
     "server:debug": "elm make src/Server.elm --output=server-app.js && nodemon server.js --config nodemon.json",
     "server:start": "node server.js",
     "start:parcel": "parcel serve index.html --no-cache",
-    "start:dev": "npm run build:init && concurrently -n \"watch,parcel,server\" -c \"green,cyan\" \"npm run start:parcel\" \"npm run server:dev\"",
```

Comment:
Is removing the watch option intended?

Reply:
My understanding is that the `watch` here is the **name** of a process that `concurrently` is running... but there are three **names** and only two **processes** (`start:parcel` and `server:dev`).

So I believe `watch` is just a leftover from some previous code.